submissions/krylenger/a-tiny-JS-world_post_OOP/index.js

```diff
@@ -0,0 +1,140 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details */
+
+const dog = {
+    type: "animal",
+    legs: 4,
+    hands: 0,
+    gender: "male",
+    friends: ["Anna", "Eugene"],
+  },
+  cat = {
+    type: "animal",
+    legs: 4,
+    hands: 0,
```

Comment:
Here we say a typical cat has 4 legs and 0 hands. In fact cats do not have hands at all.
The beaty of OOP concepts is that in class hierarchy allows super (parent) classes to have properties shared by all child classes and child classes to have properties specific to them.

Reply:
✅ Done